### PR TITLE
Fix minor bugs/changes in README.md

### PR DESCRIPTION
**Pull request by YJDave**
Solved all issues/bugs mentioned in #77

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,9 +53,9 @@ To create a loklak object you can assign the `Loklak()` object to a variable.
 eg. `l = Loklak()`
 This creates an objects whose backend loklak server is `http://loklak.org/`
 
-If you want to set this API to use your own server, you can now define it by doing
-`l = Loklak('http://192.168.192.5:9000/')` for example or pass a URL to it as
-`l = Loklak('http://loklak-super-cluster.mybluemix.net/')
+If you want to set this API to use your own server, you can now define it by doing<br />
```
Comment (singhpratyush) [inline README.md]: Since this is markdown, you should go with double space for line breaks.

File: README.md [hunk 2]
```diff
@@ -215,15 +215,16 @@ What this can do ?
...
 
-eg. `l.user('loklak_app')`
-eg. `l.user('loklak_app',1000)` - 1000 followers of `loklak_app`
-eg. `l.user('loklak_app',1000,1000)` - 1000 followers and following of `loklak_app`
-eg. `l.user('loklak_app',None,1000)` - 1000 following of `loklak_app`
+e.g.
+1. `l.user('loklak_app')`<br />
+2. `l.user('loklak_app',1000)` - 1000 followers of `loklak_app`<br />
+3. `l.user('loklak_app',1000,1000)` - 1000 followers and following of `loklak_app`<br />
+4. `l.user('loklak_app',None,1000)` - 1000 following of `loklak_app`<br />
```
Comment (singhpratyush) [inline README.md]: Please make the spaces consistent here too.

File: README.md [hunk 3]
```diff
@@ -215,15 +215,16 @@ What this can do ?
...
 
-eg. `l.user('loklak_app')`
-eg. `l.user('loklak_app',1000)` - 1000 followers of `loklak_app`
-eg. `l.user('loklak_app',1000,1000)` - 1000 followers and following of `loklak_app`
-eg. `l.user('loklak_app',None,1000)` - 1000 following of `loklak_app`
+e.g.  
+1. `l.user('loklak_app')`  
+2. `l.user('loklak_app',1000)` - 1000 followers of `loklak_app`  
+3. `l.user('loklak_app',1000,1000)` - 1000 followers and following of `loklak_app`  
+4. `l.user('loklak_app',None,1000)` - 1000 following of `loklak_app`
```
Comment (singhpratyush) [inline README.md]:
```python
# Expected
l.user('loklak_app', None, 1000)
# Current
l.user('loklak_app',None,1000)
```

**Comment by YJDave** [comment]
Thanks @singhpratyush for reviewing!!
Updated with requested changes!

File: README.md (hunk 3 above)
Comment (YJDave) [reply README.md]: Oh, yeah. Can't figure out. Thanks for suggesting.

File: README.md (hunk 1 above)
Comment (YJDave) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (YJDave) [reply README.md]: Done!

**Pull request merged**